portfolio/src/main/webapp/script.js

```diff
@@ -15,21 +15,23 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
+async function addRandomGreeting() {
+  /*const greetings =
```

Comment:
Generally, you can delete code liberally. Version control will take care of the rest (i.e. you can always restore code from history).